edge/pkg/metamanager/process.go

```diff
@@ -232,6 +301,34 @@ func (m *metaManager) processUpdate(message model.Message) {
 			resp := message.NewRespByMessage(&message, OK)
 			sendToCloud(resp)
 			return
+		case constants.ResourceTypeBackendList:
```

Comment:
anywhere we send the `ResourceTypeBackendList`？

Reply:
"backend" would not be used currently.